cmd/osmosisd/cmd/root.go

```diff
@@ -99,7 +99,31 @@ type DenomUnitMap struct {
 	Exponent uint64 `json:"exponent"`
 }
 
+type OsmosisMempoolConfig struct {
+	MaxGasWantedPerTx         string `mapstructure:"max-gas-wanted-per-tx"`
+	MinGasPriceForArbitrageTx string `mapstructure:"arbitrage-min-gas-fee"`
+	MinGasPriceForHighGasTx   string `mapstructure:"min-gas-price-for-high-gas-tx"`
+	Mempool1559Enabled        string `mapstructure:"adaptive-fee-enabled"`
+}
+
+var DefaultOsmosisMempoolConfig = OsmosisMempoolConfig{
+	MaxGasWantedPerTx:         "60000000",
+	MinGasPriceForArbitrageTx: ".1",
+	MinGasPriceForHighGasTx:   ".0025",
+	Mempool1559Enabled:        "true",
+}
```

Comment:
I wish we defined the app.toml as non string values so we could reference the config directly in txfees, but doing so would require breakage of the current app.toml string values.

Also note, this was essentially already existing, it was just defined inside one of the methods, I pulled it out so we could define defaults.